astro/src/content/docs/apis/_users-response-body.mdx

```diff
@@ -24,218 +24,20 @@ import RemovedSince from 'src/components/api/RemovedSince.astro';
-  </APIField>
-  <APIField name="users[x].twoFactor.methods[x].authenticator.codeLength" type="Integer">
-    The length of code generated by the TOTP. With the current implementation, this will always be <code>6</code>.
-  </APIField>
-  <APIField name="users[x].twoFactor.methods[x].authenticator.timeStep" type="Integer">
-    The time-step size in seconds. With the current implementation, this will always be 30.
-  </APIField>
-  <APIField name="users[x].twoFactor.methods[x].email" type="String">
-    The value of the email address for this method. Only present if <InlineField>user.twoFactor.methods<code></code>[x]<code></code>.method</InlineField> is <code>email</code>.
-  </APIField>
-  <APIField name="users[x].twoFactor.methods[x].id" type="String">
-    The unique Id of the method.
-  </APIField>
-  <APIField name="users[x].twoFactor.methods[x].lastUsed" type="Boolean">
-    <code>true</code> if this method was used most recently.
-  </APIField>
-  <APIField name="users[x].twoFactor.methods[x].method" type="String">
-    The type of this method. There will also be an object with the same value containing additional information about this method.  The possible values are:
-
-    * <code>authenticator</code>
-    * <code>email</code>
-    * <code>sms</code>
-  </APIField>
-  <APIField name="users[x].twoFactor.methods[x].mobilePhone" type="String">
-    The value of the mobile phone for this method. Only present if <InlineField>user.twoFactor.methods<code></code>[x]<code></code>.method</InlineField> is <code>sms</code>.
-  </APIField>
-  <APIField name="users[x].twoFactorDelivery" type="String" deprecated>
-    The User's preferred delivery for verification codes during a two factor login request.
-
-    The possible values are:
-
-    * <code>None</code>
-    * <code>TextMessage</code>
 
-    <RemovedSince since="1.26.0" />
-  </APIField>
-  <APIField name="users[x].twoFactorEnabled" type="Boolean" deprecated>
-    Determines if the User has two factor authentication enabled for their account or not.
-
-    <RemovedSince since="1.26.0" />
-  </APIField>
-  <APIField name="users[x].username" type="String">
-    The username of the User.
-  </APIField>
-  <APIField name="users[x].usernameStatus" type="String">
-  </APIField>
-<ModerationStatusResponse />
-
-  <APIField name="users[x].verified" type="Boolean">
-    Whether or not the User's email has been verified.
-  </APIField>
-  <APIField name="user[x].verifiedInstant" type="Long" since="1.48.0">
-    The [instant](/docs/reference/data-types#instants) that this email address was verified.
-
-    Note that this value is immutable and will only represent the first time the email was verified. If you enable re-verification on email change, this value will not change and will only represent the initial verification.
-  </APIField>
+  <UserResponseBodyBase base_field_name="users[x]" />
```

Comment:
`_user_response_body` spreads `{...props}` when including this component, but it's not doing that here. What's up with that?

Reply:
Unlike the other code path, users.mdx does not pass any props down that base needs, only `<UsersResponseBody search_result />`